internal/v1/handler_blueprints.go

```diff
@@ -276,13 +307,12 @@ func (h *Handlers) CreateBlueprint(ctx echo.Context) error {
 		}
 	}
 
-	if !blueprintNameRegex.MatchString(blueprintRequest.Name) {
-		return ctx.JSON(http.StatusUnprocessableEntity, HTTPErrorList{
-			Errors: []HTTPError{{
-				Title:  "Invalid blueprint name",
-				Detail: blueprintInvalidNameDetail,
-			}},
-		})
+	// Check blueprint rules (name and users)
+	if err := CheckBlueprintRules(ctx, &blueprintRequest, nil); err != nil {
+		if errors.Is(err, ErrBlueprintValidation) {
```

Comment:
Here you can just use `errors.As` and completely drop `ErrBlueprintValidation` because both Is and As helpers do drill down the chain of wrapped errors.